src/patternfly/demos/Tabs/examples/Tabs.md

```diff
@@ -385,7 +385,7 @@ section: components
       {{#> modal-box-body modal-box-body--attribute=(concat 'id="' modal-template--id '-modal-description"')}}
         {{#> grid grid--modifier="pf-m-gutter"}}
           {{#> grid-item}}
-            {{#> tabs tabs--id=(concat modal-template--id '-tabs') tabs--modifier="pf-m-inset-none pf-m-secondary"}}
+            {{#> tabs tabs--id=(concat modal-template--id '-tabs') tabs--IsSecondary="true" tabs--HasNoBorderBottom="true" tabs--modifier="pf-m-inset-none"}}
```

Comment:
Sorry, I'm just creating extra work 😅 You had it right the first time.

```suggestion
            {{#> tabs tabs--id=(concat modal-template--id '-tabs') tabs--IsSecondary="true" tabs--modifier="pf-m-inset-none"}}
```

Reply:
No worries! That was a nice catch either way 😄